src/Client/App.fs

```diff
@@ -32,8 +32,8 @@ module Subscriptions =
                 return true
             }
         SetARCFile =
-            fun (file, name) -> promise {
-                ApiCall.Finished(Some(file, name))
+            fun (file, name, parentId, dataMapParent) -> promise {
```

Comment:
`parentId, dataMapParent` combine these into one anonym record type as option. This will come from ARCitect so we it must be anonym record type!

Reply:
Also open an issue on ARCitect that documents this change, so we have a TODO list on ARCitect side